README.md

```diff
@@ -49,6 +49,21 @@ Try out Solidus with one-click on Heroku:
 Getting started
 ---------------
 
+Begin by making sure you have Imagemagick installed, which is required for Paperclip.
+
+* Option 1: Install using [Homebrew](https://brew.sh) (Mac only) - first, install Homebrew:
+    ```shell
+    /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+    ```
+    Then, install ImageMagick:
+    ```shell
+    brew install imagemagick
+    ```
+
+* Option 2: Install [ImageMagick](http://www.imagemagick.org/script/index.php) on Unix / Windows:
+
+  Download a ready-to-run [Unix or Windows executable](https://www.imagemagick.org/script/download.php) -OR- Install Imagemagick from [Unix or Windows Source](https://www.imagemagick.org/script/install-source.php)
```

Comment:
I would prefer to just link to the install notes of imagemagick http://imagemagick.org/script/download.php

Then this paragraph can be reduced to just a note about this dependency